Create ui-c8y-1020-2-7-asset-selector-not-emitting-correctly-the-first-selected-item-6511-graft.md (#2216)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-2-7-asset-selector-not-emitting-correctly-the-first-selected-item-6511-graft.md

---
date: ""

> title: Fixed asset selector to correctly emit the first selected item

Reviewer: ```suggestion
title: Asset selector now correctly emits the first selected item
```

> title: asset-selector not emitting correctly the first selected item (#6511) [GRAFT][release/cd] (#6581)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fixed asset selector to correctly emit the first selected item
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-59543
version: 1020.2.7
---

> asset-selector not emitting correctly the first selected item (#6511) [GRAFT][release/cd] (#6581)

Author: The Claude AI suggested to use the following body:
```suggestion
The asset selector component is used to allow users to select an asset from a list. Previously, when a user selected the first item in the list, the component did not correctly emit the selected item. This has now been fixed so that the asset selector component will always emit the selected item, including the first item in the list. This change ensures that the selected asset is properly communicated and can be used as expected.
```

> The asset selector component is used to allow users to select an asset from a list. Previously, the first item in the list was not automatically selected. This was changed, but the first selected item was not properly emitted to other components. In some cases this led to data points not showing up in the data points selector. This has now been fixed so that the asset selector component will always emit the selected item. This change ensures that the selected asset is properly communicated and depending components can be used as expected.

Reviewer: ```suggestion
The asset selector component is used to allow users to select an asset from a list. Previously, the first item in the list was not automatically selected. This had been changed, but the first selected item was not properly emitted to other components. In some cases this led to data points not showing up in the data points selector. This has now been fixed so that the asset selector component will always emit the selected item. This change ensures that the selected asset is properly communicated and depending components can be used as expected.
```

> The asset selector component is used to allow users to select an asset from a list. Previously, the first item in the list was not automatically selected. This was changed, but the first selected item was not properly emitted to other components. This led in some cases to data points not showing in the data points selector. This has now been fixed so that the asset selector component will always emit the selected item. This change ensures that the selected asset is properly communicated and depending components can be used as expected.

Reviewer: ```suggestion
The asset selector component is used to allow users to select an asset from a list. Previously, the first item in the list was not automatically selected. This was changed, but the first selected item was not properly emitted to other components. In some cases this led to data points not showing up in the data points selector. This has now been fixed so that the asset selector component will always emit the selected item. This change ensures that the selected asset is properly communicated and depending components can be used as expected.
```
